### PR TITLE
Sf fuels refactor + bole harvest litter logic fix/export frac

### PR DESCRIPTION
**Pull request by huangmy**
### Description:

This pull request aims to refactor the SPITFIRE and wood harvest logic as follows:
   1. Reorder the  fuel classes in SPITFIRE to be consistent with that in other parts of FATES;
   2. Fix a reversed logic in treating CWD contributed by directly logged trees 
   3. Add an export fraction to account for transportation losses when shipping the boles offsite

### Collaborators:

@jkshuman
@rgknox

### Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the in-code documentation .AND. (the [technical note](https://github.com/NGEET/fates-docs) .OR. the wiki) accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/NGEET/fates/blob/master/CONTRIBUTING.md) document.
- [ ] FATES PASS/FAIL regression tests were run
- [ ] If answers were expected to change, evaluation was performed and provided

### Test Results:

CTSM (or) E3SM (specify which) test hash-tag:

CTSM (or) E3SM (specify which) baseline hash-tag:

FATES baseline hash-tag:

Test Output:

### CODE REVIEW

File: biogeochem/EDPhysiologyMod.F90 [hunk 1]
```diff
@@ -1225,32 +1226,36 @@ subroutine CWD_Input( currentSite, currentPatch)
              ! Send AGB component of boles from non direct-logging activities to AGB litter pool
```
Comment (rgknox) [inline biogeochem/EDPhysiologyMod.F90]: @huangmy , if I recall correctly, we had flagged this section of code to be fixed right?  I think when c==ncwd, is when we need to calculate the trunk product, and reduce AG CWD accordingly, but we do the opposite.

**Comment by rgknox** [comment]
I reviewed the logic of adding the export fraction, and the refactor of the bole's trunk product logic, and it looks good.  I also looked through the re-assignments of the litter pool indices, but with less scrutiny.

**Comment by rgknox** [comment]
I recommend you merge this @huangmy and then submit a PR to NGEET/fates

**Comment by huangmy** [comment]
@rgknox shall I simply submit a PR from this branch to NGEET/fates? My master is identical to NGEET/fates master at this moment.

**Comment by rgknox** [comment]
That sounds great @huangmy

**Pull request merged**